src/main/java/de/clearit/kindergarten/ConfigBtn.java

```diff
@@ -0,0 +1,38 @@
+package de.clearit.kindergarten;
+
+import java.awt.Dimension;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+
+import javax.swing.JButton;
+import javax.swing.JPanel;
+
+public class ConfigBtn extends JPanel {
```

Comment:
Naja, eingentlich ist das ja kein Config*Button* mehr, sondern ein Panel mit zwei Buttons.
Also bennen es doch in ConfigPanel um.